components/api_request_helper/api_request_helper.cc

```diff
@@ -453,7 +453,7 @@ void APIRequestHelper::URLLoaderHandler::ParseSSE(
         };
 
     DVLOG(2) << "Going to call ParseJson";
-    GetDataDecoder()->ParseJson(std::move(json.data()),
```

Comment:
i'm curious how this behaved/s before and after.

`std::move(json.data())` is unusual, doesn't make sense and behaved sporadically odd.

for a moment, let's examine this: json.data() returns a `const char*`, the consumer of this pointer reads char up to the null terminator string.

how did this end up with sentence cut-offs?

Reply:
https://bravesoftware.slack.com/archives/C05NDQP03AQ/p1700249757581809?thread_ts=1699393531.992589&cid=C05NDQP03AQ